lib/puppet/face/facts.rb

```diff
@@ -87,4 +87,44 @@
       nil
     end
   end
+
+  action(:ps) do
```

Comment:
`ps` makes me think of the process status command line utility and is a little cryptic, maybe `puppet facts list` or `puppet facts show`?

Reply:
Done